R/Simulations-methods.R

```diff
@@ -978,14 +978,23 @@ setMethod("show",
       )
 
 
-
-
       ## add one reporting line
       r$report(
         "fitAtDoseMostSelected",
         "Fitted toxicity rate at dose most often selected"
       )
 
+      # Report results of additional statistics summary
+
+      if (length(list()) > 0) {
+        summary_stat_op <- unlist(object@additional_stats)
+
+        cat(
+          "Results of Additional Statistical Calculation : \n",
+          paste(names(summary_stat_op), ":", round(summary_stat_op), "\n")
```

Comment:
ok as a first version, just wondering if we should follow up in another PR to make this printing more flexible